src/libraries/System.Private.CoreLib/src/System/Reflection/Emit/Opcode.cs

```diff
@@ -45,7 +45,15 @@ internal OpCode(OpCodeValues value, int flags)
         internal bool EndsUncondJmpBlk() =>
             (m_flags & EndsUncondJmpBlkFlag) != 0;
 
-        internal int StackChange() =>
+        /// <summary>
+        /// The value of how the IL instruction changes the evaluation stack.
+        /// </summary>
+        /// <remarks>
+        /// The difference between how many elements are popped from the stack and how many are pushed onto the stack as a result of the IL instruction.
+        /// For some IL instructions like <see cref="OpCodes.Call"/> this value is not fixed and depends on the called reference signature, for example if it is
```

Comment:
You are right that the actual value is not fixed, but this method is going to return 0 in this case. It means that 0 returned by this method means either zero delta or unknown delta.

Would it make more sense to throw an exception for unknown delta instead of returning 0?

Reply:
> Would it make more sense to throw an exception for unknown delta instead of returning 0?

That would change the existing `RuntimeILGenerator` behavior, I think I should note that in that case user should not rely on this value for calculating `StackSize` nor for calculating `MaxStack`.

Reply:
> That would change the existing `RuntimeILGenerator` behavior 

Actually, it's not have to affect `RuntimeILGenerator` as we can handle the throwing in the new ILGenerator implementation, though not sure such special casing is worth, don't you think adding a `NOTE` in the doc could be enough?

Reply:
> Actually, it's not have to affect `RuntimeILGenerator` as we can handle the throwing in the new ILGenerator implementation, 

I take it back, handling the throwing in the new ILGenerator doesn't make sense 😄